lib/token.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+
+package hwsc;
+
+enum Permission {
+    ADMIN = 0;
+    USER = 1;
+}
+
+message Token {
```

Comment:
when chrome gets authorized and chrome wants to request a service like `modifyUser`, chrome has to resend this exact Token message to `app-gateway-svc`?

Reply:
A Token has to be sent in all transactions.